db/models.py

```diff
@@ -42,11 +55,100 @@ def __str__(self) -> str:
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=CinemaHall, on_delete=models.CASCADE, related_name="cinema_hall"
     )
     movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=Movie, on_delete=models.CASCADE, related_name="movie"
```

Comment:
Do not change the `related_name`, they were defined correctly.

Reply:
The comment regarding `related_name` is not resolved, return the previous names.